homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -225,57 +240,77 @@ def _async_setup_keypads(
     hass: HomeAssistant,
     config_entry_id: str,
     bridge: Smartbridge,
-    bridge_device: dict[str, Any],
+    bridge_device: dict[str, str | int],
 ) -> LutronKeypadData:
     """Register keypad devices (Keypads and Pico Remotes) in the device registry."""
 
     device_registry = dr.async_get(hass)
 
-    bridge_devices = bridge.get_devices()
-    bridge_buttons = bridge.buttons
+    bridge_devices: dict[str, dict[str, str | int]] = bridge.get_devices()
```

Comment:
I had so much trouble following this code so I added some more typing to make sure I didn't screw it up